src/app/reports/utility/immunised.html

```diff
@@ -7,12 +7,12 @@ <h5 class="pull-left">Average Immunized </h5>
   <h3 class="pull-right">| {{utilityCtrl.kpiAgg.ave}}</h3>
 </div>
 <div class="btn btn-success btn-lg clearfix">
-  <h5 class="pull-left">Highest Immunized <br/> ({{utilityCtrl.rounds[utilityCtrl.kpiValues[0][0]]}})</h5>
-  <h3 class="pull-right">| {{utilityCtrl.kpiValues[0][1]}}</h3>
+  <h5 class="pull-left">Highest Immunized <br/> ({{utilityCtrl.rounds[utilityCtrl.kpiValues[utilityCtrl.kpiValues.length -1][0]]}})</h5>
```

Comment:
Sure, but what is its value and why did it change from the first element? It's not clear to me when reading the code as-is (without running it).